<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20349" started-at="2020-07-10T11:43:23 IST" finished-at="2020-07-10T11:43:43 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20349" started-at="2020-07-10T11:43:23 IST" finished-at="2020-07-10T11:43:43 IST">
      <class name="pages.BaseClass$BaseTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:pages.BaseClass$BaseTest@4fb64261]" name="setup" is-config="true" duration-ms="20056" started-at="2020-07-10T11:43:23 IST" finished-at="2020-07-10T11:43:43 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
      </class> <!-- pages.BaseClass$BaseTest -->
      <class name="tests.MyshaadiTestCase">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:tests.MyshaadiTestCase@42607a4f]" name="beforeClass" is-config="true" duration-ms="0" started-at="2020-07-10T11:43:43 IST" finished-at="2020-07-10T11:43:43 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:tests.MyshaadiTestCase@42607a4f]" name="beforeMethod" is-config="true" duration-ms="5" started-at="2020-07-10T11:43:43 IST" finished-at="2020-07-10T11:43:43 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.MyshaadiTestCase.shaadi()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="shaadi()[pri:0, instance:tests.MyshaadiTestCase@42607a4f]" name="shaadi" duration-ms="12" started-at="2020-07-10T11:43:43 IST" finished-at="2020-07-10T11:43:43 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at pages.AppLaunch.clickOnLogin(AppLaunch.java:13)
at tests.MyshaadiTestCase.shaadi(MyshaadiTestCase.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shaadi -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:tests.MyshaadiTestCase@42607a4f]" name="afterMethod" is-config="true" duration-ms="3" started-at="2020-07-10T11:43:43 IST" finished-at="2020-07-10T11:43:43 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:tests.MyshaadiTestCase@42607a4f]" name="afterClass" is-config="true" duration-ms="1" started-at="2020-07-10T11:43:43 IST" finished-at="2020-07-10T11:43:43 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="FAIL" signature="teardown()[pri:0, instance:tests.MyshaadiTestCase@42607a4f]" name="teardown" is-config="true" duration-ms="96" started-at="2020-07-10T11:43:43 IST" finished-at="2020-07-10T11:43:43 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at pages.BaseClass.teardown(BaseClass.java:82)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:623)
at org.testng.TestRunner.afterRun(TestRunner.java:853)
at org.testng.TestRunner.run(TestRunner.java:594)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- tests.MyshaadiTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
